app/templates/components/nav-bar.hbs

```diff
@@ -1,31 +1,85 @@
+    {{#if showNavBarHeader}}
+      <div class="navbar-header navbar-right info-actions hidden-xs">
+        {{#link-to 'signup-step1' class='btn btn-primary btn-sm btn-login'}}
+          ¡Contrata ya!
+        {{/link-to}}
+        <a href="{{dashboardUrl}}" class="btn btn-default btn-sm btn-login" target="_blank">
+          Inicia Sesión
+        </a>
+      </div>
+    {{/if}}
+  </div> -->
+  <!-- <div class="container-fluid info hidden-xs visible-sm">
+    {{nav-bar-phones}}
+    {{#if showNavBarHeader}}
+      <div class="navbar-header navbar-right info-actions hidden-xs">
+        {{#link-to 'signup-step1' class='btn btn-primary btn-sm btn-login'}}
+          ¡Contrata ya!
+        {{/link-to}}
+        <a href="{{dashboardUrl}}" class="btn btn-default btn-sm btn-login" target="_blank">
+          Inicia Sesión
+        </a>
+      </div>
+    {{/if}}
+  </div> -->
+  <div class="container-fluid">
+    <div class="navbar-header logo">
+      {{#if showNav}}
+        <button type="button" class="navbar-toggle collapsed" aria-expanded="false" aria-controls="navbar" onClick={{action 'toggleCollapsed'}}>
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+      {{/if}}
+      <a href="{{homeUrl}}" class="navbar-brand">
+        <img src="/assets/images/main-logo.png" alt="GurúComm" />
+      </a>
+    </div>
     {{#if showNav}}
-      <button type="button"
-        class="navbar-toggle collapsed"
-        aria-expanded="false"
-        aria-controls="navbar" onClick={{action 'toggleCollapsed'}}>
-        <span class="sr-only">Toggle navigation</span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-      </button>
+      {{#bs-collapse id="navbar" collapsed=navBarIsCollapsed class="collapse navbar-collapse navbar-right"}}
+        <ul class="nav navbar-nav navbar-left">
+          <li>
+            {{#link-to 'sign-up'}}
+              ¿Cómo funciona?
+            {{/link-to}}
+          </li>
+          <li>
+            {{#link-to 'sign-up'}}
+              Trabaja desde casa
+            {{/link-to}}
+          </li>
+          <li>
+            {{#link-to 'sign-up'}}
+              Nuestro proyecto
+            {{/link-to}}
+          </li>
+          <li>
+            {{#link-to 'sign-up'}}
+              Contacto
+            {{/link-to}}
+          </li>
+          <li>
+            {{#link-to 'sign-up'}}
+              Quiero mi número
+            {{/link-to}}
+          </li>
+          <li class="visible-xs">
+           <a href="{{dashboardUrl}}" target="_blank">
+             Inicia Sesión
+           </a>
+         </li>
+          <!-- TODO: restore  -->
+          <!-- <li>
+            {{#link-to 'partner'}}
+              Únete a GurúComm
+            {{/link-to}}
+          </li> -->
```

Comment:
@PerlaElizabethAguilar puedes poner solo los link to que van en la página por favor?